internal/restinterface/resthelper/client/httphelper/httphelper.go

```diff
@@ -25,7 +25,8 @@ import (
 
 var client *http.Client
 
-type HttpHelper struct{}
+//HTTPHelper struct
```

Comment:
```suggestion
// HTTPHelper struct
```

@nitu-s-gupta Same trivial suggestion from me. 😄

Reply:
DOne